client/extensions/woocommerce/app/reviews/reviews-list.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { range } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import EmptyContent from 'components/empty-content';
+import { fetchReviews } from 'woocommerce/state/sites/reviews/actions';
+import {
+	areReviewsLoading,
+	areReviewsLoaded,
+	getReviews,
+	getTotalReviews
+} from 'woocommerce/state/sites/reviews/selectors';
+import { getReviewsCurrentPage, getReviewsCurrentSearch } from 'woocommerce/state/ui/reviews/selectors';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Pagination from 'components/pagination';
+import ReviewCard from './review-card';
+import ReviewsFilterNav from './reviews-filter-nav';
+import { updateCurrentReviewsQuery } from 'woocommerce/state/ui/reviews/actions';
+
+class ReviewsList extends Component {
+	componentDidMount() {
+		const { siteId, currentStatus } = this.props;
+		const query = {
+			page: 1,
+			search: '',
+			status: currentStatus,
+		};
+		this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, search: '' } );
+		if ( siteId ) {
+			this.props.fetchReviews( siteId, query );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		const hasAnythingChanged = (
+			newProps.currentPage !== this.props.currentPage ||
+			newProps.currentSearch !== this.props.currentSearch ||
+			newProps.currentStatus !== this.props.currentStatus ||
+			newProps.siteId !== this.props.siteId
+		);
+		if ( ! newProps.siteId || ! hasAnythingChanged ) {
+			return;
+		}
+
+		const query = {
+			page: newProps.currentPage,
+			search: newProps.currentSearch,
+			status: newProps.currentStatus,
+		};
+		if ( newProps.currentSearch !== this.props.currentSearch ) {
+			this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, status: 'any' } );
+			query.page = 1;
+			query.status = 'any';
+		} else if ( newProps.currentStatus !== this.props.currentStatus ) {
+			this.props.updateCurrentReviewsQuery( this.props.siteId, { page: 1, search: '' } );
+			query.page = 1;
+			query.search = '';
+		}
+
+		if ( '' !== query.search ) {
+			query.status = 'any';
+		}
+
+		this.props.fetchReviews( newProps.siteId, query );
+	}
+
+	renderPlaceholders = () => {
+		return range( 5 ).map( ( i ) => {
+			return (
+				<Card key={ i } className="reviews__card">
+					<div className="reviews__placeholder">
+					</div>
+				</Card>
+			);
+		} );
+	}
+
+	renderNoContent = () => {
+		const { currentSearch, currentStatus, translate } = this.props;
+		let emptyMessage = translate( 'No pending reviews.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **105** times:
`translate( 'Pending Review' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-review-list-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).